run-tests.sh

```diff
@@ -7,8 +7,14 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-pydocstyle invenio_iiif tests docs && \
-isort -rc -c -df && \
-check-manifest --ignore ".travis-*" && \
-sphinx-build -qnNW docs docs/_build/html && \
+# Quit on errors
+set -o errexit
+
+# Quit on unbound symbols
+set -o nounset
+
+pydocstyle invenio_iiif tests docs
```

Comment:
`pydocstyle` shouldn't be tested at this level anymore. We test it via `pytest`.